.github/workflows/basic.yml

```diff
@@ -344,78 +344,39 @@ jobs:
       matrix:
         include:
           - name: Defaults
-            test_cmd: ./CppUTest_tests -r
+            make_target: test_defaults
           - name: STDC++ disabled
-            make_args: >
-              CPPUTEST_USE_STD_CPP_LIB=N
-              extensions
+            make_target: test_disable_std_cpp
           - name: Memory Leak Detection disabled
-            make_args: >
-              CPPUTEST_USE_MEM_LEAK_DETECTION=N
-              extensions
-          - name: Memory Leak Detection disabled and STD C++ disabled
-            make_args: >
-              CPPUTEST_USE_MEM_LEAK_DETECTION=N
-              CPPUTEST_USE_STD_CPP_LIB=Y
-              extensions
+            make_target: test_disable_memory_leak_detection
+          - name: Disable Leak Detection and STD C++
+            make_target: test_disable_std_cpp_and_leak_detection
           - name: debug disabled
-            make_args: >
-              CPPUTEST_ENABLE_DEBUG=N
-              extensions
+            make_target: test_disable_debug
           - name: overridden CXXFLAGS and CFLAGS and CPPFLAGS
-            make_args: >
-              CLFAGS=""
-              CXXFLAGS=""
-              CPPFLAGS="-Iinclude"
+            make_target: test_override_flags
           - name: without Standard C library includes
-            make_args: >
-              CPPUTEST_USE_STD_C_LIB=N
-              CPPUTEST_ADDITIONAL_CPPFLAGS=-DCPPUTEST_CHAR_BIT=8
-              all_no_tests
+            make_target: test_disable_std_c
           - name: different TARGET_PLATFORM
-            make_args: TARGET_PLATFORM=real_platform
-          - name: overridden CXXFLAGS and CFLAGS and memory leak and STDC++ disabled
-            make_args: >
-              CLFAGS=""
-              CXXFLAGS=""
-              CPPFLAGS="-Iinclude -DCPPUTEST_STD_CPP_LIB_DISABLED -DCPPUTEST_MEM_LEAK_DETECTION_DISABLED"
+            make_target: test_target_platform
+          - name: override flags and disable memory leak and STDC++
+            make_target: test_override_and_disable
           - name: Examples
-            make_args: examples
+            make_target: test_examples
           - name: JUnit Output
-            test_cmd: ./CppUTest_tests -ojunit
-          - name: gcov
-            make_args: >
-              CPPUTEST_USE_GCOV=Y
-              everythingInstall
-            test_cmd: >
-              make -f Makefile_using_MakefileWorker gcov &&
-              make -f Makefile_CppUTestExt gcov &&
-              make -C examples gcov
+            make_target: test_junit_output
+          # - name: gcov
+          #   make_target: test_gcov
```

Comment:
Something is broken with this: https://github.com/cpputest/cpputest/actions/runs/3666346383/jobs/6198003504